helper_crates/vtable/CHANGELOG.md

```diff
@@ -1,6 +1,10 @@
 # Changelog
 All notable changes to this crate will be documented in this file.
 
+## [0.1.6]
```

Comment:
0.1.5 was not yet released, so no need to bump the version at this point.

Reply:
Ahh, thanks - will revert then.